### PR TITLE
Improve type handling when passing parameters to create flow run mutation

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Fixes an issue where the editor would incorrectly coerce single-item arrays to scalar values and non-strings to strings in JSON editing mode on the run page.

### CODE REVIEW

File: src/components/CustomInputs/DictInput.vue [hunk 1]
```diff
@@ -182,6 +191,7 @@ export default {
           Object.fromEntries(v.map(entry => [entry.key, entry.value]))
         )
       } else {
+        // console.log()
```
Comment (zhen0) [inline src/components/CustomInputs/DictInput.vue:194]: Could we take this out?

**Review by zhen0 — CHANGES_REQUESTED** [review]
One tiny nitpick from me but the type handling improvements look good.

**Comment by znicholasbrown** [comment]
@zangell44 - we haven't traditionally tested these components cause the tests ended up being very brittle but I agree this is something we should test in the future

**Review by zhen0 — APPROVED** [review]
Thanks for the change @znicholasbrown

**Pull request merged**